src/HasDependencies.php

```diff
@@ -139,9 +140,10 @@ protected function applyRulesForChildFields($childField)
     /**
      * Validate action fields
      * Overridden using ActionController & ActionRequest by modifying routes
+     * @param  \Laravel\Nova\Http\Requests\ActionRequest  $request
      * @return void
      */
-    public function validateFields() {
+    public function validateFields(ActionRequest $request = null) {
```

Comment:
Why the fallback to null here? I tested a few of my usages of resources and a new action using the same code just without the null default and I didn't get an error.

Reply:
In nova versions <3.21.0, this method might be called without any arguments, so defaulting to null would solve this

Reply:
hmm good point there. Good safety net then.